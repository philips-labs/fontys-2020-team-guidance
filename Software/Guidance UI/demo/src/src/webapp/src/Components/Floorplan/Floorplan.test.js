import React from 'react';
import { render, cleanup, fireEvent} from '@testing-library/react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Floorplan from "./Floorplan";

afterEach(cleanup);

it('renders correctly react-test-renderer', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<Floorplan />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
});