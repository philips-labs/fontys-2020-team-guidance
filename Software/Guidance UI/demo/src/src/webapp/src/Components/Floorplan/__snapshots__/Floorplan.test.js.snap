// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly react-test-renderer 1`] = `
<div
  id="floorplan-container"
>
  <div
    className="floorplanNode"
    id="user"
    style={
      Object {
        "backgroundColor": "green",
        "border": "1px solid white",
        "borderRadius": "45px",
        "left": "0px",
        "padding": "8px",
        "position": "absolute",
        "top": "0px",
      }
    }
  />
  <img
    alt=""
    className="img"
    draggable="false"
    id="floorplan-container-image"
    onMouseDown={[Function]}
    onTouchStart={[Function]}
    onWheel={[Function]}
    style={Object {}}
  />
</div>
`;
